android-agent/src/main/kotlin/io/opentelemetry/android/agent/session/SessionIdGenerator.kt

```diff
@@ -5,18 +5,6 @@
 
 package io.opentelemetry.android.agent.session
 
-import io.opentelemetry.api.trace.TraceId
-import java.util.Random
-
-interface SessionIdGenerator {
+internal fun interface SessionIdGenerator {
```

Comment:
Oh I think this was public on purpose. There have been users who previously wanted to provide their own session id values....

Reply:
I don't see any public-facing API that allows setting a custom `SessionIdGenerator` or `SessionStorage` - what's the expected usage?